run-build.sh

```diff
@@ -154,7 +154,7 @@ export DOTNET_MULTILEVEL_LOOKUP=0
 # Install a stage 0
 INSTALL_ARCHITECTURE=$ARCHITECTURE
 archlower="$(echo $ARCHITECTURE | awk '{print tolower($0)}')"
-if [[ $archlower == 'arm'* ]]; then
+if [[ $archlower == 'arm'* ]] || [[ $archlower == 'arm64'* ]]; then
```

Comment:
@sdmaclea 
Doesn't "$archlower == 'arm'*" cover both "arm" and "arm64"?

Reply:
It does.  It was a syntax I was unfamiliar with, so I missed it.  Done.